ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivity.kt

```diff
@@ -49,6 +60,28 @@ internal class PaywallActivity : ComponentActivity(), PaywallListener {
         }
     }
 
+    @Parcelize
+    private data class SdkConfigArgs(
```

Comment:
We could use `PurchasesConfiguration` directly if we make it parcelable. We can ignore the `ExecutorService` using `@IgnoredOnParcel`. See [the docs](https://developer.android.com/kotlin/parcelize#skip_properties_from_serialization). 

The upside of using `PurchasesConfiguration` directly is that we don't accidentally add more properties that aren't parcelable, or at least it forces us to think about that.

Reply:
I think we also would need to ignore the `Context` right? And then we would need to get the application context again, which I'm not sure if it would be easy to do from there...

Reply:
Ah yes, we'd also need to ignore the `Context`! It would be difficult to get the Application Context from `PurchasesConfiguration` indeed. Nevermind haha.